plugins/recaptcha/class.recaptcha.plugin.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+/**
+ * @copyright 2010-2016 Vanilla Forums Inc
+ * @license Proprietary
+ */
+
+$PluginInfo['recaptcha'] = array(
+    'Name' => 'Recaptcha Support',
+    'Description' => "Add recaptcha validation to signups.",
+    'Version' => '0.1',
+    'MobileFriendly' => true,
+    'Author' => "Tim Gunter",
+    'AuthorEmail' => 'tim@vanillaforums.com',
+    'AuthorUrl' => 'http://www.vanillaforums.com'
+);
+
+/**
+ * Recaptcha Validation
+ *
+ * This plugin adds recaptcha validation to signups.
+ *
+ * Changes:
+ *  0.1        Development
+ *
+ * @author Tim Gunter <tim@vanillaforums.com>
+ * @package vanilla
+ */
+class RecaptchaPlugin extends Gdn_Plugin {
+
+    /**
+     * reCAPTCHA private key
```

Comment:
Pretty sure the format for property doc blocks is `/** @var string reCAPTCHA private key. */`